docs/api/background-tasks.mdx

```diff
@@ -71,11 +71,11 @@ curl --location 'https://api.permit.io/v2/projects/default/envs/staging/copy/asy
 
 ## Waiting for Background tasks
 
-You can also wait for the background task to complete by using the `wait` query parameter.
+You can also wait for the background task to be completed by using the `wait` query parameter.
 
-The wait parameter specify the maximum time to wait for the task to complete in seconds.
+The `wait` parameter specifies the maximum time to wait for the task to complete in seconds.
 By default, `wait=0` will return immediately with the current status of the task.
-As soon as the task complete, the API will respond the result of the task.
+As soon as the task is complete, the API will respond to the result of the task.
```

Comment:
Also became weird sentence. When the task is completed, the API will respond immediately with the result of the task.